include/stl2/view/drop.hpp

```diff
@@ -0,0 +1,117 @@
+// cmcstl2 - A concept-enabled C++ standard library
+//
+//  Copyright Casey Carter
+//  Copyright Eric Niebler
+//  Copyright Christopher Di Bella
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/caseycarter/cmcstl2
+//
+#ifndef STL2_VIEW_DROP_HPP
+#define STL2_VIEW_DROP_HPP
+
+#include <stl2/detail/fwd.hpp>
+#include <stl2/detail/meta.hpp>
+#include <stl2/detail/iterator/concepts.hpp>
+#include <stl2/detail/non_propagating_cache.hpp>
+#include <stl2/detail/range/access.hpp>
+#include <stl2/detail/range/concepts.hpp>
+#include <stl2/detail/view/view_closure.hpp>
+#include <stl2/view/all.hpp>
+#include <stl2/view/view_interface.hpp>
+
+STL2_OPEN_NAMESPACE {
+	namespace ext {
+		template <View R>
+		class drop_view : public view_interface<drop_view<R>> {
+			using D = iter_difference_t<iterator_t<R>>;
+		public:
+			drop_view() = default;
+
+			constexpr drop_view(R base, D count)
+				: base_(std::move(base)),
+				  count_(count)
+			{}
+
+			template <ViewableRange O>
+			requires
+				_ConstructibleFromRange<R, O>
+			constexpr drop_view(O&& o, D count)
+				: base_(view::all(std::forward<O>(o))),
+				  count_(count)
+			{}
+
+			constexpr R base() const { return base_; }
+
+			constexpr auto begin() requires !SimpleView<R> { return begin_impl(*this); }
+			constexpr auto begin() const requires Range<R const>
```

Comment:
Despite that Eric and I are both east `const`-ers, cmcstl2 uses `const` west - at least for occurrences in the interface - to follow the conventions of the IS and minimize differences between the declarations as they appear in the implementation vs. the spec. I can live with the east `const` usage in the function bodies and interfaces of private functions (I hate `const` west enough to live with the inconsistency.)